overlord/overlord_test.go

```diff
@@ -682,7 +710,8 @@ func (ovs *overlordSuite) TestEnsureLoopPruneDoesntAbortShortlyAfterStartOfOpera
 
 	// start the loop that runs the prune ticker
 	o.Loop()
-	time.Sleep(1500 * time.Millisecond)
+	w.tick(10)
```

Comment:
Similar question as before, why 10 (and not some other value), is this significant? I wonder if 2 would work too (if my understanding in my earlier comment is correct it should).